test/test_cpp_extensions_jit.py

```diff
@@ -47,12 +47,14 @@ class TestCppExtensionJIT(common.TestCase):
     def setUp(self):
         # cpp extensions use relative paths. Those paths are relative to
         # this file, so we'll change the working directory temporarily
+        super().setUp()
```

Comment:
might it be clearer to put the comments right next to what they're referring to?
```suggestion
        super().setUp()
        # cpp extensions use relative paths. Those paths are relative to
        # this file, so we'll change the working directory temporarily
```